docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -297,22 +309,26 @@ Like regular jobs, asset jobs can be placed on [schedules](/concepts/partitions-
 
 ## Testing
 
-Since a function decorated with <PyObject object="asset" /> acts as the op for a software defined asset. You can write unit tests for this function in the same way you would a standard [op](/concepts/testing#testing-ops).
+Since a function decorated with <PyObject object="asset" /> acts as the op for a software defined asset, you can write unit tests for this function in the same way you would write unit tests for a standard op. See this section on [op testing](/concepts/testing#testing-ops) for some background information.
+
+Similar to the `@op` decorator, the `@asset` decorator can slightly modify the call signiture of the decorated function. However, the new signiture closely mirrors the signature of the decorated function, and in some cases, they match exactly.
```

Comment:
In what cases does the decorator modify the call signature of the decorated function?

Reply:
yeah that's misworded - i meant that you can have a `context` arg that will be provided by the decorator. fixing